### PR TITLE
FIRECERT-1945: Import fixtures from Config modules to FCS' fixture 

### PR DESCRIPTION
**Pull request by PreethiMaai**

### CODE REVIEW

File: cypress/TestCases/README.md [hunk 1]
```diff
@@ -28,3 +28,9 @@ Here are some examples of tags for different categories:
 * Tags for module: @accessibility
 * Tags for sdk: @coreSDK, @manageSDK
 * Tags for communicationModes: @sdk, @transport
```
Comment (neeradanelxsi) [inline cypress/TestCases/README.md:30]: We should not add anything related to configModule inside FCS .  If required, that can be added in configModule readme

File: cypress/fixtures/README.md [hunk 2]
```diff
@@ -5,3 +5,26 @@ Fixtures are designed to load a fixed set of data located in a file.
 ## Required data need to be updated before running the feature files
 ### Advertising: 
  - APPBUNDLEID - It contains the corresponding app ID and should be updated in `configModule/testData/modules/advertising.json` for content validation.
```
Comment (neeradanelxsi) [inline cypress/fixtures/README.md:7]: We should not add anything related to configModule inside FCS . If required, that can be added in configModule readme

File: cypress/fixtures/README.md [hunk 3]
```diff
@@ -5,3 +5,26 @@ Fixtures are designed to load a fixed set of data located in a file.
...
+
+## Override concepts:
```
[the remark this answers is not in the document]
Comment (PreethiMaai) [reply cypress/fixtures/README.md:5]: Corrected

File: cypress/TestCases/README.md (hunk 1 above)
Comment (PreethiMaai) [reply cypress/TestCases/README.md:30]: Removed configModule related documentations

File: cypress/fixtures/README.md [hunk 4]
```diff
@@ -2,6 +2,24 @@
...
- - APPBUNDLEID - It contains the corresponding app ID and should be updated in `configModule/testData/modules/advertising.json` for content validation.
+## Override concepts:
+-  If a key is present in FCS but not in the ConfigModule, the FCS key will be used without any overrides.
+-  If a key is present in the ConfigModule but not in FCS, The key from the ConfigModule is copied over and used.
+-  If a key is present in both FCS and the ConfigModule, the ConfigModule's key will override or merge with precedence given to the ConfigModule.
+
+## Directory strucute:
+Within the `cypress/fixtures` folder we have the following sub-folders:
+- fireboltCalls: Contains FireboltCall keys which is used to make firebolt calls to the 1st party app or 3rd party app.
+- fireboltMocks: Contains FireboltMock keys which is used to override the default response or set the responses.
```
Comment (neeradanelxsi) [inline cypress/fixtures/README.md:13]: - objects: 
  **validationObjects**: Contains json files where required validation json objects are present.  
  **errorObjects**: Contains error objects for validating negative scenario parameters. 
  It contains two json files as : errorContent.json and errorSchemaObject.json. Error response is validated against the schema present in errorSchemaObject.json . errorContent.json contains error Messages and error Codes for different error types.
  **moduleReqId**: Contains scenario name and requirement ID for test cases.

File: cypress/fixtures/README.md [hunk 5]
```diff
@@ -2,6 +2,24 @@
...
+- modules: Contains json files in the format of <moduleName.json> where required content/parameters are present.
```
Comment (neeradanelxsi) [inline cypress/fixtures/README.md:14]: image
below 2 folders are missing

**Review by kschrief — APPROVED** [review]

File: cypress/fixtures/README.md (hunk 4 above)
Comment (PreethiMaai) [reply cypress/fixtures/README.md:13]: Updated

File: cypress/fixtures/README.md (hunk 5 above)
Comment (PreethiMaai) [reply cypress/fixtures/README.md:14]: Included these 2 folders in readme.

**Review by neeradanelxsi — APPROVED** [review]

**Review by neeradanelxsi — APPROVED** [review]

**Pull request merged**